### PR TITLE
check for restart on atomic

### PR DESCRIPTION
**Pull request by mjudeikis**
fix atomic host storage

### CODE REVIEW

File: roles/openshift_storage_glusterfs/tasks/glusterfs_common.yml [hunk 1]
```diff
@@ -313,7 +313,10 @@
   - glusterfs_storageclass or glusterfs_s3_deploy
 
 - include_tasks: glusterblock_deploy.yml
-  when: glusterfs_block_deploy
+  when:
```
Comment (vrutkovs) [inline roles/openshift_storage_glusterfs/tasks/glusterfs_common.yml]: Tox doesn't like trailing space here

Comment (jarrpa) [reply roles/openshift_storage_glusterfs/tasks/glusterfs_common.yml]: @mjudeikis Always run tox before submitting a PR. :) I say this as someone who often forgets to do that.

Comment (mjudeikis) [reply roles/openshift_storage_glusterfs/tasks/glusterfs_common.yml]: dam.  But are we fine now to just ignore atomic host? (i think we dont have other options :/ )

Comment (jarrpa) [reply roles/openshift_storage_glusterfs/tasks/glusterfs_common.yml]: It's what we've been doing, unfortunately. I need to sit down with my team and figure out what the strategy on Atomic is, I just haven't had the time.

Comment (mjudeikis) [reply roles/openshift_storage_glusterfs/tasks/glusterfs_common.yml]: sorted... sorry, too many contexts switches these days...

**Pull request merged**